ledfx/api/websocket.py

```diff
@@ -275,6 +275,20 @@ def audio_stream_config_handler(self, message):
                 message.get("data"),
             )
         )
+        try:
+            ACTIVE_AUDIO_STREAM._bits = int(message.get("bits"))
```

Comment:
I think this will return a default value of None and therefore the except code will not fire to set default to 12 bits

Shouldn't it be get("bits", 12)

or just message["bits"]

Reply:
Yep, good catch, will fix that.